### PR TITLE
PLATFORM-7774 | EmbedVideo update to MediaWiki 1.39

### PR DESCRIPTION
**Pull request by MrVanosh**
https://fandom.atlassian.net/browse/PLATFORM-7774
https://gitlab.com/hydrawiki/extensions/EmbedVideo/-/merge_requests/155

### CODE REVIEW

File: EmbedVideo.hooks.php [hunk 1]
```diff
@@ -72,16 +72,15 @@ class EmbedVideoHooks {
 	 * @return void
 	 */
 	public static function onExtension() {
-		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions;
-
-		$config = MediaWikiServices::getInstance()->getConfigFactory()->makeConfig('main');
+		global $wgEmbedVideoDefaultWidth, $wgMediaHandlers, $wgFileExtensions,
+			$wgEmbedVideoEnableAudioHandler, $wgEmbedVideoEnableVideoHandler, $wgEmbedVideoAddFileExtensions;
```
Comment (emkarcinos) [inline EmbedVideo.hooks.php:76]: Lets get all those values from `MainConfig`. If possible, rewrite the hook to a hook handler https://www.mediawiki.org/wiki/Manual:Hooks

**Review by emkarcinos — CHANGES_REQUESTED** [review]
.

File: EmbedVideo.hooks.php (hunk 1 above)
Comment (MrVanosh) [reply EmbedVideo.hooks.php:76]: Ok

Comment (MrVanosh) [reply EmbedVideo.hooks.php:76]: Not possible to get these values from MainConfig as calling MediaWikiServices here is deprecated and function must be static to call in extension registration so dependency injection also is unavailable and it's impossible to change configuration values with MainConfig. In Wikimedia extensions these configurations variables are also used by global. Example: (https://github.com/wikimedia/mediawiki-extensions-CirrusSearch/blob/master/includes/Hooks.php)

**Review by emkarcinos — APPROVED** [review]

**Pull request merged**